Documentation/CRDs/Cluster/external-cluster/external-cluster.md

```diff
@@ -210,42 +219,42 @@ If not installing with Helm, here are the steps to install with manifests.
     rook-ceph-external   /var/lib/rook                162m   Connected   HEALTH_OK
     ```
 
-2. Verify the creation of the storage class depending on the rbd pools and filesystem provided.
+1. Verify the creation of the storage class depending on the rbd pools and filesystem provided.
     `ceph-rbd` and `cephfs` would be the respective names for the RBD and CephFS storage classes.
 
     ```console
     kubectl -n rook-ceph-external get sc
     ```
 
-3. Then you can now create a [persistent volume](https://github.com/rook/rook/tree/master/deploy/examples/csi) based on these StorageClass.
+1. Then you can now create a [persistent volume](https://github.com/rook/rook/tree/master/deploy/examples/csi) based on these StorageClass.
 
 ### Connect to an External Object Store
 
-Create the [external object store CR](https://github.com/rook/rook/blob/master/deploy/examples/object-external.yaml) to configure connection to external gateways.
+Create the [external object store CR](https://github.com/rook/rook/blob/master/deploy/examples/external/object-external.yaml) to configure connection to external gateways.
 
 ```console
-    cd deploy/examples
+    cd deploy/examples/external
     kubectl create -f object-external.yaml
 ```
 
 Consume the S3 Storage, in two different ways:
 
 1. Create an [Object store user](https://github.com/rook/rook/blob/master/deploy/examples/object-user.yaml) for credentials to access the S3 endpoint.
 
-```console
-    cd deploy/examples
-    kubectl create -f object-user.yaml
-```
+   ```console
+      cd deploy/examples
+      kubectl create -f object-user.yaml
+   ```
 
-2. Create a [bucket storage class](https://github.com/rook/rook/blob/master/deploy/examples/storageclass-bucket-delete.yaml) where a client can request creating buckets and then create the [Object Bucket Claim](https://github.com/rook/rook/blob/master/deploy/examples/object-bucket-claim-delete.yaml), which will create an individual bucket for reading and writing objects.
+1. Create a [bucket storage class](https://github.com/rook/rook/blob/master/deploy/examples/storageclass-bucket-delete.yaml) where a client can request creating buckets and then create the [Object Bucket Claim](https://github.com/rook/rook/blob/master/deploy/examples/object-bucket-claim-delete.yaml), which will create an individual bucket for reading and writing objects.
```

Comment:
we should add symlinks for all of these, and update the document to use external documents.